docs/victoriametrics/changelog/CHANGELOG.md

```diff
@@ -18,9 +18,12 @@ See also [LTS releases](https://docs.victoriametrics.com/lts-releases/).
 
 ## tip
 
+** update note 1: change vmselect to vmstorage cluster RPC method from `metricNamesUsageStats_v1` to `metricNamesUsageStats_v2`
```

Comment:
please be more descriptive from user PoV. 
For user protocol change matters not. What matters is that vmselect of newer version won't be able to "talk" to older vmstorage for this specific API. This should be part of update note